### ISSUE TITLE
commando: missing fts4 sqlite module error

### ISSUE THREAD

**Issue by twhiteman** [Komodo/KomodoEdit#698]
Komodo IDE, version 9.3.0, build 87940, platform macosx.
Built on Thu Oct 8 02:25:32 2015.

At startup (or first open of commando), I see this in error log:
```
Exception in thread Index Builder:
Traceback (most recent call last):
  File "/Applications/Komodo IDE 9.app/Contents/Frameworks/Python.framework/Versions/2.7/lib/python2.7/threading.py", line 808, in __bootstrap_inner
    self.run()
  File "/Applications/Komodo IDE 9.app/Contents/Frameworks/Python.framework/Versions/2.7/lib/python2.7/threading.py", line 761, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/Applications/Komodo IDE 9.app/Contents/Resources/extensions/scope-docs@activestate.com/components/koScopeDocs.py", line 148, in _preload
    indexer = Indexer(main=self)
  File "/Applications/Komodo IDE 9.app/Contents/Resources/extensions/scope-docs@activestate.com/components/koScopeDocs.py", line 427, in __init__
    self.createDb()
  File "/Applications/Komodo IDE 9.app/Contents/Resources/extensions/scope-docs@activestate.com/components/koScopeDocs.py", line 474, in createDb
    ''')
  File "/Applications/Komodo IDE 9.app/Contents/Resources/extensions/scope-docs@activestate.com/components/koScopeDocs.py", line 212, in query
    return self.cursor.execute(query)
OperationalError: no such module: fts4
```

**Comment by Naatan** [comment]
Seems our packaged Python doesn't support Sqlite4 on OSX, hopefully we can simply package it.

**Comment by Naatan** [comment]
This has been resolved on the IDE branch and should be in the next nightly.

**Comment by twhiteman** [comment]
Nice - and now they just released SQLite 3.9 with FTS5 :P

**Comment by Naatan** [comment]
Well after spending a day learning how NOT to do it I'm confident updating won't be a problem :p For now we'll stick with what we have though.
